### PR TITLE
Meta: Add a few more e-mail addresses to the .mailmap file

### PR DESCRIPTION
**Pull request by gunnarbeutner**

### CODE REVIEW

File: .mailmap [hunk 1]
```diff
@@ -4,12 +4,27 @@ Brian Gianforcaro <bgianf@serenityos.org> <b.gianfo@gmail.com>
...
+<filmroellchen@serenityos.org> <28656157+kleinesfilmroellchen@users.noreply.github.com>
 Lenny Maiorani <lenny@serenityos.org> <lenny@colorado.edu>
 Kenneth Myhra <kennethmyhra@serenityos.org> <kennethmyhra@gmail.com>
 networkException <networkexception@serenityos.org> <git@nwex.de>
 networkException <networkexception@serenityos.org> <github@nwex.de>
 Thomas Voss <mail@thomasvoss.com> <thomasvoss@live.com>
+<kling@serenityos.org> <awesomekling@gmail.com>
+<kling@serenityos.org> <ankl@hms.se>
+<liavalb@gmail.com> <51659396+supercomputer7@users.noreply.github.com>
+Liav A <liavalb@gmail.com>
```
Comment (linusg) [inline .mailmap:21]: Can't we shorten this?

```suggestion
Liav A <liavalb@gmail.com> <51659396+supercomputer7@users.noreply.github.com>
```

File: .mailmap [hunk 2]
```diff
@@ -4,12 +4,27 @@ Brian Gianforcaro <bgianf@serenityos.org> <b.gianfo@gmail.com>
...
+<akaster@serenityos.org> <andrewdkaster@gmail.com>
+<idan.horowitz@gmail.com> <idan.horowitz@ibm.com>
+<leon.a@serenityos.org> <37629766+Hendiadyoin1@users.noreply.github.com>
+Hendiadyoin1 <leon.a@serenityos.org> <leon2002.la@gmail.com>
```
Comment (linusg) [inline .mailmap:25]:
```suggestion
Hendiadyoin1 <leon.a@serenityos.org> <37629766+Hendiadyoin1@users.noreply.github.com>
<leon.a@serenityos.org> <leon2002.la@gmail.com>
```

File: .mailmap [hunk 3]
```diff
@@ -4,12 +4,27 @@ Brian Gianforcaro <bgianf@serenityos.org> <b.gianfo@gmail.com>
...
+Jesse Buhagiar <jooster669@gmail.com> <Jooster669@gmail.com>
+Jesse Buhagiar <jooster669@gmail.com> <jesse.buhagiar@student.rmit.edu.au>
```
Comment (linusg) [inline .mailmap:27]:
```suggestion
<jooster669@gmail.com> <jesse.buhagiar@student.rmit.edu.au>
```

**Review by linusg — COMMENTED** [review]
Not entirely familiar with the mailmap, but generally it seems we should list the canonical address first, and not duplicate display names?

File: .mailmap (hunk 1 above)
Comment (gunnarbeutner) [reply .mailmap]: 👍

File: .mailmap (hunk 2 above)
Comment (gunnarbeutner) [reply .mailmap]: 👍

File: .mailmap (hunk 3 above)
Comment (gunnarbeutner) [reply .mailmap:25]: Without specifying the name explicitly we end up with this - so I'm going to keep that as is:

```
   8 Author: Jesse <jooster669@gmail.com>
 146 Author: Jesse Buhagiar <jooster669@gmail.com>
```

**Pull request merged**